cloud/linode/loadbalancers.go

```diff
@@ -246,55 +249,36 @@ func (l *loadbalancers) getNodeBalancerDeviceId(ctx context.Context, firewallID,
 	return 0, false, nil
 }
 
-// updateNodeBalancerFirewall updates the firewall attached to the nodebalancer
-//
-// Firewall is updated in 3 scenario's:
-// - Add new firewall to the nodeBalancer
-// - Update firewall attached to the nodebalancer
-// - Remove firewall attached
-func (l *loadbalancers) updateNodeBalancerFirewall(ctx context.Context, service *v1.Service, nb *linodego.NodeBalancer) error {
+func (l *loadbalancers) updateFWwithID(ctx context.Context, service *v1.Service, nb *linodego.NodeBalancer) error {
```

Comment:
Please add docs here + maybe expand the name, no need to shorten firewall to FW.